web/ui/react-app/src/pages/targets/ScrapePoolContent.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, useEffect, useState } from 'react';
+import { getColor, Target } from './target';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { Badge, Table } from 'reactstrap';
+import TargetLabels from './TargetLabels';
+import styles from './ScrapePoolPanel.module.css';
+import { formatRelative } from '../../utils';
+import { now } from 'moment';
+import TargetScrapeDuration from './TargetScrapeDuration';
+import EndpointLink from './EndpointLink';
+
+const columns = ['Endpoint', 'State', 'Labels', 'Last Scrape', 'Scrape Duration', 'Error'];
+const initialNumberOfTargetsDisplayed = 50;
+
+interface ScrapePoolContentProps {
+  targets: Target[];
+}
+
+export const ScrapePoolContent: FC<ScrapePoolContentProps> = ({ targets }) => {
+  const [items, setItems] = useState<Target[]>(targets.slice(0, 50));
+  const [index, setIndex] = useState<number>(initialNumberOfTargetsDisplayed);
+  const [hasMore, setHasMore] = useState<boolean>(targets.length > initialNumberOfTargetsDisplayed);
+
+  useEffect(() => {
+    setItems(targets.slice(0, initialNumberOfTargetsDisplayed));
+    setHasMore(targets.length > initialNumberOfTargetsDisplayed);
+  }, [targets]);
+
+  const fetchMoreData = () => {
+    if (items.length === targets.length) {
+      setHasMore(false);
+    } else {
+      const newIndex = index + initialNumberOfTargetsDisplayed;
+      setIndex(newIndex);
+      setItems(targets.slice(0, newIndex));
+    }
+  };
+
+  return (
+    <InfiniteScroll
+      next={fetchMoreData}
+      hasMore={hasMore}
+      loader={<h4>loading...</h4>}
+      dataLength={items.length}
+      height={items.length > 25 ? '75vh' : ''}
+    >
+      <Table className={styles.table} size="sm" hover>
```

Comment:
In general with this new table design, it's good by itself (though striped could still be good for readability of long lines), but we have other pages with similar tables that still use the old design. So we either have to change all of them at once or keep this one at the old design for now.

Reply:
mmm let's keep the old design, there are already enough changes in this PR.